pkg/agent/config/node_config.go

```diff
@@ -25,10 +25,11 @@ const (
 	// Invalid ofport_request number is in range 1 to 65,279. For ofport_request number not in the range, OVS
 	// ignore the it and automatically assign a port number.
 	// Here we use an invalid port number "0" to request for automatically port allocation.
-	AutoAssignedOFPort = 0
-	DefaultTunOFPort   = 1
-	HostGatewayOFPort  = 2
-	UplinkOFPort       = 3
+	// AutoAssignedOFPort = 0
+	DefaultTunOFPort  = 1
+	HostGatewayOFPort = 2
+	UplinkOFPort      = 3
+	HostOFPort        = 4
 	// 0xfffffffe is a reserved port number in OpenFlow protocol, which is dedicated for the Bridge interface.
 	BridgeOFPort = 0xfffffffe
```

Comment:
Is `BridgeOFPort` needed in the newest code?

Reply:
Removed.